originals/c.yaml

```diff
@@ -120,6 +120,23 @@
   names:
   - Doukutsu Monogatari
 
+- name: Celeste Classic
+  names:
+  - Celeste
+  external:
+    wikipedia: Celeste (video game)
+  platform:
+    - Linux
+    - macOS
```

Comment:
this doesn't exist
Please check the schema; the build errors for reference: https://github.com/opengaming/osgameclones/runs/4057615513?check_suite_focus=true